### PR TITLE
Spawn case

### PR DESCRIPTION
**Pull request by gabeduke**
Translate dict keys in command list and event.command_name to lowercase for standardized read.
* This helps when bot alias is `.` as phone keyboards will auto uppercase the first letter after a period
* example: `. define word` will be auto-cased to `. Define word` which breaks the spawn command

### CODE REVIEW

**Comment by gabeduke** [comment]
Much cleaner thanks for the suggestion. Tested and updated the requirements.

File: requirements.txt [hunk 1]
```diff
@@ -20,3 +20,4 @@ slackclient >=0.16 # slackrtm (includes websockets)
 textblob
 python_dateutil # gitlab sink
 telepot == 7.1 # telegram sync, newer versions doesn't support Python3.4.1
+CaseInsensitiveDict # spawn
```
Comment (pleasantone) [inline requirements.txt]: not needed

File: hangupsbot/plugins/spawn.py [hunk 2]
```diff
@@ -65,20 +65,22 @@
 import os
 import logging
 import asyncio
+from CaseInsensitiveDict import CaseInsensitiveDict
```
Comment (pleasantone) [inline hangupsbot/plugins/spawn.py]: From collections import ...

**Comment by gabeduke** [comment]
Excellent thanks! I'm still new to python & programming so I'll be interested to see what is the proper implementation.

**Comment by pleasantone** [comment]
Closed in favor of #730

**Pull request closed without merging**